Assets/Scripts/Multiplayer/Photon/PhotonManager.cs

```diff
@@ -153,6 +153,19 @@ public async Task<bool> JoinRoom(RoomCreateData roomCreateData)
 
             if (result.Ok)
             {
+                // Verify if the room is actually full
```

Comment:
Is this count separate from the count of the room itself? Is there a possibility of a race condition if a room with max=4 has 3 people, and then two try to join at the same time?

Reply:
_Runner.SessionInfo.PlayerCount accurately reflects the current number of players in the room based on the latest state from the Fusion NetworkRunner. It's the closest approximation available.

Reply:
OK StartGame(args) result.ok reflects the fact that the game started this is a quick fix which should hold for beta .. however i need to look better in the matchmaking to get the proper way